ui/src/main/js/page/audit/AuditPage.js

```diff
@@ -10,7 +10,7 @@ import NotificationTypeLegend from 'page/audit/NotificationTypeLegend';
 import AuditDetails from 'page/audit/Details';
 import CheckboxInput from 'common/component/input/CheckboxInput';
 import * as DescriptorUtilities from 'common/util/descriptorUtilities';
-import ConfigurationLabel from 'common/component/ConfigurationLabel';
+import PageHeader from '../../common/components/navigation/PageHeader';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import StatusMessage from 'common/component/StatusMessage';
 import '../../../css/audit.scss';
```

Comment:
We missed this import we should fix it.

Reply:
Looking into this one, it throws an error if the absolute path is removed

Reply:
Fixed!